src/app/rpmostree-pkg-builtins.c

```diff
@@ -127,11 +127,9 @@ pkg_change (RPMOSTreeSysroot *sysroot_proxy,
     }
   else if (!opt_reboot)
     {
-      const char *sysroot_path;
-
-
-      sysroot_path = rpmostree_sysroot_get_path (sysroot_proxy);
+      /* install/uninstall currently always results in a new deployment */
 
```

Comment:
Hey, slowly trying to do code review here(mainly asking questions :p for familiarization ) , hopefully you won't mind :). So I noticed there are a few places in the patch having an extra line between the comments and the code, is that intended? Thanks :). 

P.S: Not sure if it is the right thing to do for a review, if I did anything wrong, feel free to let me know :D

Reply:
Yep, definitely a valid point to raise I think.

Reply:
Good catch, no good reason for those spaces!